cinema/models.py

```diff
@@ -41,10 +45,19 @@ class Movie(models.Model):
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(null=True, upload_to="movies")
```

Comment:
The `upload_to` parameter for the `image` field in the `Movie` model should use the `create_custom_path` method to generate a custom path for uploaded images. Consider updating it to use `Movie.create_custom_path`.